app/domain/output_document.rb

```diff
@@ -20,13 +20,16 @@ def attendee_name
   end
 
   def attendee_address
+    attendee_country = Countries.uk?(self.attendee_country) ? nil : self.attendee_country
```

Comment:
It is a bit weird, but I thought it'd be weirder to have an `attendee_country` method that returns nil.

As discussed, we should make the individual address methods private as a separate PR.